### PR TITLE
Improvements to udev logging and related code

### PR DESCRIPTION
**Pull request by keszybz**
The goal is to make `udevadm test` output easier to read.

### CODE REVIEW

File: src/libsystemd/sd-device/device-util.h [hunk 1]
```diff
@@ -37,7 +37,7 @@
                 sd_device *_d = (device);                               \
                 int _level = (level), _error = (error);                 \
                                                                         \
-                if (_d && _unlikely_(log_get_max_level() >= _level))    \
+                if (_d && _unlikely_(log_get_max_level() >= LOG_PRI(_level))) \
```
Comment (yuwata) [inline src/libsystemd/sd-device/device-util.h:40]: The change itself is ok. But I have a question, why this change is related to `SYNTHETIC_ERRNO()`?

Comment (keszybz) [reply src/libsystemd/sd-device/device-util.h:40]: You're right, my commit message makes no sense. I got the level and errno fields crossed.

Comment (keszybz) [reply src/libsystemd/sd-device/device-util.h:40]: Updated with the commit message rewrite.

Comment (yuwata) [reply src/libsystemd/sd-device/device-util.h:40]: Thanks.

**Comment by keszybz** [comment]
```
autopkgtest [21:28:58]: test logind: [-----------------------
 * try-restarting systemd-logind
 * daemon is started
 * loginctl succeeds
 * XDG_SESSION_ID=
1:name=systemd:/user.slice/user-1000.slice/session-1.scope
0::/user.slice/user-1000.slice/session-1.scope
 * process is in session cgroup
-- Logs begin at Thu 2020-07-16 03:18:42 UTC. --
Jul 16 21:14:29 autopkgtest systemd[1]: /lib/systemd/system/systemd-logind.service:58: Unknown system call group, ignoring: @system-service
Jul 16 21:28:58 autopkgtest systemd[1]: Stopping User Login Management...
Jul 16 21:28:58 autopkgtest systemd[1]: systemd-logind.service: Succeeded.
Jul 16 21:28:58 autopkgtest systemd[1]: Stopped User Login Management.
Jul 16 21:28:59 autopkgtest systemd[1]: Starting User Login Management...
Jul 16 21:29:00 autopkgtest systemd-logind[32706]: Watching system buttons on /dev/input/event0 (Power Button)
Jul 16 21:29:00 autopkgtest systemd-logind[32706]: Watching system buttons on /dev/input/event1 (AT Translated Set 2 keyboard)
Jul 16 21:29:00 autopkgtest systemd-logind[32706]: New seat seat0.
Jul 16 21:29:00 autopkgtest systemd[1]: Started User Login Management.
Jul 16 21:29:00 autopkgtest systemd-logind[32706]: New session 1 of user ubuntu.
Fake Lid Switch: /dev/input/event4
Jul 16 21:29:03 autopkgtest systemd-logind[32706]: Watching system buttons on /dev/input/event4 (Fake Lid Switch)
   waiting for suspend, 30s remaining...
   waiting for suspend, 25s remaining...
   waiting for suspend, 20s remaining...
   waiting for suspend, 15s remaining...
   waiting for suspend, 10s remaining...
   waiting for suspend, 5s remaining...
   waiting for suspend, 0s remaining...
closing lid did not cause suspend
autopkgtest [21:29:35]: test logind: -----------------------]
autopkgtest [21:29:35]: test logind:  - - - - - - - - - - results - - - - - - - - - -
logind               FAIL non-zero exit status 1
```
Presumably not related.
